templates/addons/aurora/postgresql-cf.yml

```diff
@@ -105,46 +105,58 @@ Resources:
   AuroraSecret:
     Type: AWS::SecretsManager::Secret
     Properties:
-      Name: !Sub '/${App}/${Env}/${Name}/${DBTableName}'
-      Description: !Sub Aurora PostgreSQL main user secret for ${AWS::StackName}
+      Name: !Sub '/${App}/${Env}/${Name}'
+      Description: !Sub Aurora main user secret for ${AWS::StackName}
       GenerateSecretString:
         SecretStringTemplate: '{"username": "postgres"}'
         GenerateStringKey: "password"
         ExcludePunctuation: true
         IncludeSpace: false
         PasswordLength: 16
-  
+
+  {{- if .ParameterGroup}}
+
+  #   DBClusterParameterGroup:
+  #     Type: 'AWS::RDS::DBClusterParameterGroup'
+  #     Properties:
+  #       Description: !Ref 'AWS::StackName'
+  #       Family: 'aurora-postgresql10'
+  #       Parameters:
+  #         client_encoding: 'UTF8'
+  {{- else}}
+
   DBClusterParameterGroup:
     Type: 'AWS::RDS::DBClusterParameterGroup'
     Properties:
       Description: !Ref 'AWS::StackName'
       Family: 'aurora-postgresql10'
       Parameters:
         client_encoding: 'UTF8'
+  {{- end}}
 
-  AuroraDBCluster:
-    Type: 'AWS::RDS::DBCluster'
-    Properties:
-      MasterUsername:
-        !Join [ "", [ "{{resolve:secretsmanager:", !Ref AuroraSecret, ":SecretString:username}}" ]]
-      MasterUserPassword:
-        !Join [ "", [ "{{resolve:secretsmanager:", !Ref AuroraSecret, ":SecretString:password}}" ]]
-      DatabaseName: !Ref DBTableName
-      Engine: aurora-postgresql # aurora-mysql
-      EngineVersion: '10.7'
-      EngineMode: serverless
-      EnableHttpEndpoint: true
-      StorageEncrypted: true
-      KmsKeyId: !Ref AuroraKMSCMK
-      DBClusterParameterGroupName: !Ref DBClusterParameterGroup
-      DBSubnetGroupName: !Ref DBSubnetGroup
-      VpcSecurityGroupIds:
-        - !Ref DBClusterSecurityGroup
-      ScalingConfiguration:
-        AutoPause: true
-        MinCapacity: !Ref DBMinCapacity
-        MaxCapacity: !Ref DBMaxCapacity
-        SecondsUntilAutoPause: !Ref DBAutoPauseSeconds
+  {{logicalIDSafe .Name}}:
+  Type: 'AWS::RDS::DBCluster'
+  Properties:
+    MasterUsername:
+      !Join [ "", [ "{{resolve:secretsmanager:", !Ref AuroraSecret, ":SecretString:username}}" ]]
+    MasterUserPassword:
+      !Join [ "", [ "{{resolve:secretsmanager:", !Ref AuroraSecret, ":SecretString:password}}" ]]
+    DatabaseName: !Ref InitialDBName
+    Engine: {{.Engine}}
+    #      EngineVersion: '10.7'
```

Comment:
We should probably not comment out the Engine Version, no? Since there are only some engine versions that work with aurora serverless we should be explicit about it.

Reply:
I think there are only one engine version that supports serverless for each of the engine (mysql5.6, mysql5.7, postgresql). It probably won't work if the customers change the value to a version that doesn't support serverless for that engine. 

Or are we having this field for informational purpose? So that they are aware which version is being used?